src/main/java/org/infinispan/tutorial/db/LocationWeatherQueryInitializer.java

```diff
@@ -0,0 +1,15 @@
+package org.infinispan.tutorial.db;
+
+import org.infinispan.protostream.SerializationContextInitializer;
+import org.infinispan.protostream.annotations.AutoProtoSchemaBuilder;
+import org.infinispan.tutorial.data.LocationWeather;
+
+@AutoProtoSchemaBuilder(
+      includeClasses = {
+            LocationWeather.class
+      },
+      schemaFileName = "weather.proto",
+      schemaFilePath = "proto/",
+      schemaPackageName = "org.infinispan.tutorial.data")
+public interface LocationWeatherQueryInitializer extends SerializationContextInitializer {
```

Comment:
I'd suggest naming this ```LocationWeatherSchema```

Reply:
```@AutoProtoSchemaBuilder``` is not processed because the protostream-processor dependency is missing from pom